app/javascript/packs/application.js

```diff
@@ -8,7 +8,6 @@ import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
 require("jquery")
-require ("jquery-ui")
```

Comment:
why you remove this line

Reply:
because i cannot delete user if i have this line.
if i click delete user, it will direct to user profile.
after i delete this line, then i can delete user.